src/main/java/com/greenfox/dramacsoport/petclinicbackend/controllers/appUser/AppUserController.java

```diff
@@ -1,35 +1,33 @@
 package com.greenfox.dramacsoport.petclinicbackend.controllers.appUser;
 
-import com.greenfox.dramacsoport.petclinicbackend.exeptions.DeletionException;
 import com.greenfox.dramacsoport.petclinicbackend.services.appUser.AppUserService;
 import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
 
 import java.security.Principal;
 
 @RequiredArgsConstructor
-@Controller
+@RestController
+@RequestMapping("api/v1/user")
 public class AppUserController {
 
     private final AppUserService appUserService;
     private final Logger logger = LoggerFactory.getLogger(AppUserController.class);
 
-    @DeleteMapping("/user/delete")
-    public ResponseEntity<?> deleteUser(Principal user) {
-        try {
-            logger.info("Deleting user: {}", user.getName());
-            return new ResponseEntity<>(appUserService.deleteUser(user.getName()), HttpStatus.OK);
-        } catch (DeletionException e) {
-            logger.error("Error while deleting user: {}", user.getName(), e);
-            return new ResponseEntity<>(e.getMessage(), HttpStatus.CONFLICT);
-        } catch (Exception e) {
-            logger.error("Error while deleting user", e);
-            return new ResponseEntity<>(e.getMessage(), HttpStatus.INTERNAL_SERVER_ERROR);
-        }
+    @SneakyThrows
+    @DeleteMapping("/{id}")
+    public ResponseEntity<?> deleteUser(@PathVariable Long id, Principal user) {
+        String userEmail = user.getName();
+
+        logger.info("Deleting user: {}", user.getName());
```

Comment:
You don't need to create a variable here just to have a reference to the username (which is the email you know 😄 ), but if you have, you should/can use it. 
```suggestion
        String userEmail = user.getName();

        logger.info("Deleting user: {}", userEmail);
```